test/e2e/validation_test.go

```diff
@@ -254,3 +261,41 @@ func filterSSMatrix(mat *types.ComMatrix) (*types.ComMatrix, error) {
 
 	return &types.ComMatrix{Matrix: res}, nil
 }
+
+func extractNotUsedAndMissingEPS(csvContent string) (*types.ComMatrix, *types.ComMatrix, error) {
+	plusLinesCSV := filterCSVPlusMinusLines(csvContent, "+")
+	minusLines := filterCSVPlusMinusLines(csvContent, "-")
+
+	notUsedEPSMat, err := types.ParseToComDetailsList([]byte(plusLinesCSV), types.FormatCSV)
```

Comment:
unused instead of notUsed.

Reply:
i use the same name as before what it was